src/opencl_ansible_fmt_plug.c

```diff
@@ -0,0 +1,343 @@
+/*
+ * This software is Copyright (c) 2018 Dhiru Kholia, Copyright (c) 2018 magnum,
+ * Copyright (c) 2017 Jim Fougeron, and Copyright (c) 2013 Lukas Odzioba <ukasz
+ * at openwall dot net> and it is hereby released to the general public under
+ * the following terms:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ */
+
+#ifdef HAVE_OPENCL
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_opencl_ansible;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_opencl_ansible);
+#else
+
+#include <stdint.h>
+#include <string.h>
+
+#include "misc.h"
+#include "arch.h"
+#include "common.h"
+#include "formats.h"
+#include "options.h"
+#include "hmac_sha.h"
+#include "ansible_common.h"
+#include "opencl_common.h"
+
+#define FORMAT_LABEL            "ansible-opencl"
+#define ALGORITHM_NAME          "PBKDF2-SHA256 HMAC-SHA256 OpenCL"
```

Comment:
@kholia I just now realized that this was incorrect, it should have been `"PBKDF2-SHA256 OpenCL HMAC-SHA256"`, with anything CPU-side after the "OpenCL". It's fixed already so no sweat but I'd like you to be adamant with this in the future because that's an easy way for us to see where we can improve formats.

Reply:
Thanks for pointing this out. I will pay attention to this from now on. I was feeling rusty earlier this morning.